docs/source/example_notebooks/dowhy_example_effect_of_memberrewards_program.ipynb

```diff
@@ -256,7 +256,7 @@
     },
     {
      "data": {
```

Comment:
can you avoid making changes to the "data" field?
you can do by checking out the remote version of this file and only changing the groupby line.